### PR TITLE
add meshexpansion with gateway support

### PR DESCRIPTION
**Pull request by irisdingbj**
Please provide a description for what this PR is for.

https://github.com/istio/istio/issues/15427

[ ] Configuration Infrastructure
[ ] Docs
[X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: install/kubernetes/helm/istio/example-values/values-istio-meshexpansion-gateways.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+global:
+  # Provides dns resolution for global services
+  podDNSSearchNamespaces:
+  - global
+  - "{{ valueOrDefault .DeploymentMeta.Namespace \"default\" }}.global"
```
Comment (sdake) [inline install/kubernetes/helm/istio/example-values/values-istio-meshexpansion-gateways.yaml]: I am prtty sure ethis oine should as of now be purged throughout all example0-values yaml files. (line 5)

Comment (irisdingbj) [reply install/kubernetes/helm/istio/example-values/values-istio-meshexpansion-gateways.yaml]: Removed line 5.

**Comment by irisdingbj** [comment]
@sdake ^^^ Thanks!

**Pull request merged**